packages/bootstrap/docs/Bootstrap.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from "react"
+import Container from "../index"
+
+export default {
+  title: "KaizenApp Container (React)",
+  parameters: {},
+  component: Container,
+}
+
+export const DefaultContainer = () => (
+  <Container>
+    <div>Wrap your app in container to ensure it is consistent</div>
```

Comment:
Might be nice to use a `Header` component so we can demonstrate fonts loading?

Reply:
It might be nice to have stories that don't include this as a default decorator, and one story with it explicitly added, and one without, to demonstrate font loading, CSS normalizing, etc